jetbrains-core/tst/software/aws/toolkits/jetbrains/services/codewhisperer/CodeWhispererConstantsTest.kt

```diff
@@ -0,0 +1,16 @@
+// Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.codewhisperer
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import software.aws.toolkits.jetbrains.services.codewhisperer.util.CodeWhispererConstants
+
+class CodeWhispererConstantsTest {
+    @Test
+    fun `is codewishperer help uri has correct doc link`() {
+        assertThat(CodeWhispererConstants.CODEWHISPERER_LOGIN_HELP_URI)
+            .isEqualTo("https://docs.aws.amazon.com/toolkit-for-jetbrains/latest/userguide/setup-credentials.html")
+    }
```

Comment:
i dont think this type of test is really useful

Reply:
Though a better test I can think of is to check the action done when clicking on the help icon or the link will open the corresponding url